discovery_engine_core/bindings/src/types/uuid.rs

```diff
@@ -0,0 +1,135 @@
+// Copyright 2022 Xayn AG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, version 3.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+//! FFI functions for handling `Uuid`
+
+use std::ptr;
+
+use uuid::Uuid;
+
+/// Creates a new UUID based on this byes (~ `[u8; 16]`).
```

Comment:
```suggestion
/// Creates a new UUID based on this bytes (~ `[u8; 16]`).
```

Reply:
wups, another word which needs to go on the spell checkers flagged list. As I probably never want to write `byes` in source code.